pyproject.toml

```diff
@@ -22,7 +22,8 @@ uvicorn = "*"
 types-toml = "*"
 numpy = "*"
 json-repair = "*"
-playwright = "*"
+browsergym = "*" # integrate browsergym as the browsing interface
```

Comment:
As far as I understand you only need a dependency on `browsergym-core`.

`browsergym` is a meta-package that includes additional benchmarks with extra dependencies, `browsergym-webarena`, `browsergym-workarena` and `browsergym-miniwob`.

Reply:
@frankxu2004

Reply:
Hi @gasse Thanks for the comment! We are also interested in evaluating agents in the OpenDevin platform on web agent benchmarks such as WebArena, WorkArena and Miniwob++, so we might just leave it as-is for now!